server/pulp/server/db/model/__init__.py

```diff
@@ -365,6 +365,34 @@ def _write_pem_file(self, config_key, path):
                 else:
                     pem_file.write(self.config[config_key])
 
+    def _set_scratchpad_entry(self, key, value):
```

Comment:
can we re-use already available methods? https://github.com/pulp/pulp/blob/2-master/server/pulp/plugins/conduits/mixins.py#L264
here's the example on how we work with the importer's scratchpad  https://github.com/pulp/pulp_rpm/blob/2-master/plugins/pulp_rpm/plugins/importers/yum/sync.py#L506

Reply:
@ggainey, @ipanova is right. We already have mixins that provide this. Sorry I led youthe wrong way.

Reply:
@ggainey @ipanova: the new helper methods should probably just move the mixin. The existing methods don't enough.

Reply:
We have the ImporterScratchpadMixin - but at this point in the code-flow, we're dealing the Importer db-model directly, which doesn't know anything about the ImporterScratchpadMixin. That's why this code relies directly on having the helpers in the Importer model - it's all it has to work with. @ipanova @dkliban , guidance on the right way to accomplish what we want here, given the limitations of the code path?

Reply:
Conduit is-an ImportScratchPadMixin, and available in the flows we care about - moved the helpers to ImportScratchPadMixin